README.md

```diff
@@ -0,0 +1,24 @@
+# Rust crate to build custom gitlab runners
```

Comment:
I don't think I usually see the header being a project description line?

Reply:
So looking at a bunch of different creates the first header is just the name of hte project or some logo (hyper, axume, tracing, anyhow ¯\_(°ペ)_/¯, etc).  So that's something we can follow. Though it's not a change introduced by these patches, so i'll probably do that as a follow-up to get this train out of the way :)